docs/topics/serialization.txt

```diff
@@ -347,6 +347,37 @@ again a mapping with the key being name of the field and the value the value:
 
 Referential fields are again represented by the PK or sequence of PKs.
 
+.. _custom-serialization-formats:
+
+Custom serialization formats
+----------------------------
+
+In addition to the above formats, you can create your own serialization format.
+
+First, define a ``Serializer`` and a ``Deserializer`` class. These can override
+existing serialization format classes::
+
+    from django.core import serializers
+
+
+    class Serializer(serializers.python.Serializer): ...
+
+
+    class Deserializer(serializers.python.Deserializer): ...
```

Comment:
@sarahboyce 
Awesome job! 
IMHO, I don't think custom deserialization quite fits into the  `Custom serialization formats`. Plus, adding a custom serializer to the `SERIALIZATION_MODULES` is a convenient way that only applies to serializers but not such a thing for a custom deserializer. So it doesn't seem cohesive to me.

Reply:
I think this is how folks are supposed to do it? `get_deserializer` loads the serializers defined by this setting, and then gets the `Deserializer` from it. So this is how you define both a custom serializer and deserializer

Reply:
Yes, you're right about the usage and how `get_deserializer` returns and I was confused. I wanted to suggest a direct mention of custom deserialization in the section title. Plus, regarding 
`First, define a ``Serializer`` and a ``Deserializer`` class. `
having a custom Deserialization doesn't raise any error in the custom module unless deserialization for that custom format gets called. Maybe it indicates it's necessary (Not a big deal!).

Reply:
> having a custom Deserialization doesn't raise any error in the custom module unless deserialization for that custom format gets called. Maybe it indicates it's necessary (Not a big deal!).

I'm not quite sure what the suggestion is, do you think this needs a docs warning? Feel free to push up updates :+1:

Reply:
Got it. I'll wait for other reviewers' comments. If the docs get no comment, it means it's clear enough.

Reply:
I think we should put an example of Serializer and Deserializer with minimal but working code, and not empty.

Reply:
Yeah I agree, a more realistic example would be beneficial. @AMK9978 do you mind pulling something together?

Reply:
@pauloxnet @sarahboyce 
I added a minimal functioning csv serializer and deserializer. Although due to the flat nature of csv, it requires a bit more code, I think it is quite useful as an example.